Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -932,25 +937,33 @@ def genS(fragment,howMuch):
 hiAlca2017 = {'--conditions':'auto:phase1_2017_realistic', '--era':'Run2_2017_pp_on_XeXe'}
 hiAlca2018 = {'--conditions':'auto:phase1_2018_realistic', '--era':'Run2_2018'}
 hiAlca2018_ppReco = {'--conditions':'auto:phase1_2018_realistic_hi', '--era':'Run2_2018_pp_on_AA'}
+hiAlca2021_ppReco = {'--conditions':'auto:phase1_2021_realistic_hi', '--era':'Run3_pp_on_PbPb'}
 
 
 hiDefaults2011=merge([hiAlca2011,{'--scenario':'HeavyIons','-n':2}])
 hiDefaults2015=merge([hiAlca2015,{'--scenario':'HeavyIons','-n':2}])
 hiDefaults2017=merge([hiAlca2017,{'-n':2}])
 hiDefaults2018=merge([hiAlca2018,{'--scenario':'HeavyIons','-n':2}])
 hiDefaults2018_ppReco=merge([hiAlca2018_ppReco,{'-n':2}])
+hiDefaults2021_ppReco=merge([hiAlca2021_ppReco,{'-n':2}])
 
 steps['HydjetQ_B12_5020GeV_2011']=merge([{'-n':1,'--beamspot':'RealisticHI2011Collision'},hiDefaults2011,genS('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
 steps['HydjetQ_B12_5020GeV_2015']=merge([{'-n':1,'--beamspot':'RealisticHICollisionFixZ2015'},hiDefaults2015,genS('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
 steps['HydjetQ_MinBias_XeXe_5442GeV_2017']=merge([{'-n':1},hiDefaults2017,gen2017('Hydjet_Quenched_MinBias_XeXe_5442GeV_cfi',U2000by1)])
 steps['HydjetQ_B12_5020GeV_2018']=merge([{'-n':1},hiDefaults2018,gen2018prod('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
 steps['HydjetQ_B12_5020GeV_2018_ppReco']=merge([{'-n':1},hiDefaults2018_ppReco,gen2018hiprod('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
+steps['HydjetQ_B12_5020GeV_2021_ppReco']=merge([{'-n':1},hiDefaults2021_ppReco,gen2021hiprod('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
 
 steps['QCD_Pt_80_120_13_HI']=merge([hiDefaults2018_ppReco,gen2018hiprod('QCD_Pt_80_120_13TeV_TuneCUETP8M1_cfi',Kby(9,150))])
 steps['PhotonJets_Pt_10_13_HI']=merge([hiDefaults2018_ppReco,gen2018hiprod('PhotonJet_Pt_10_13TeV_TuneCUETP8M1_cfi',Kby(9,150))])
-steps['ZMM_13_HI']=merge([hiDefaults2018_ppReco,gen2018hiprod('ZMM_13TeV_TuneCUETP8M1_cfi',Kby(18,100))])
 steps['ZEEMM_13_HI']=merge([hiDefaults2018_ppReco,gen2018hiprod('ZEEMM_13TeV_TuneCUETP8M1_cfi',Kby(18,300))])
 
+steps['QCD_Pt_80_120_14_HI_2021']=merge([hiDefaults2021_ppReco,gen2021hiprod('QCD_Pt_80_120_14TeV_TuneCUETP8M1_cfi',Kby(9,150))])
+steps['PhotonJets_Pt_10_14_HI_2021']=merge([hiDefaults2021_ppReco,gen2021hiprod('PhotonJet_Pt_10_14TeV_TuneCUETP8M1_cfi',Kby(9,150))])
+#steps['ZEEMM_13_HI_2021']=merge([hiDefaults2021_ppReco,gen2021hiprod('ZEEMM_13TeV_TuneCUETP8M1_cfi',Kby(18,300))])
```

Comment:
@mandrenguyen I thought we agreed to have ZEEMM not separated. Could you add corresponding GEN fragment and update this part?

Reply:
@chayanit What is the logic of running them together?

Reply:
@mandrenguyen what is the logic to run them separate? We used to run them together in 2018 no?

Reply:
@chayanit  The logic is to keep them in sync easily with the high lumi pp workflows.  I don't see why we should add an extra layer of complexity to anyone analyzing the output, as well as add additional gen fragments to maintain.  If you're worried about the extra processing, we can cut the number of events in half.

Reply:
@mandrenguyen ZEEMM has been used as HI workflow since 2018 and now you said it's adding extra layer of complexity by just selecting Z from muon or electron? If you want to keep them in sync with high lumi pp, you will need to update all workflows to TuneCP5.

Reply:
Yes, was the one who made the PR to add ZEEMM IIRC.  Can we not be stuck with it forever, please?

Reply:
So please remove ZEEMM line you commented out completely if you wanna move to separate workflows. How about the Tuning? Do we stay with the old tuning for HI?

Reply:
Ok, will do this afternoon, thanks!

Reply:
@chayanit  I just pushed the change to CP5 for the ZEE and ZMM wfs (and removed the commented out wf).  I don't see CP5 fragments for the QCD and gamma-jet.  Did you want me to migrate those as well?

Reply:
@mandrenguyen it would be nice to have everything in the same setup. Thanks!

Reply:
I have added the CP5 fragments and updated the relevant workflows